fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryWrapper.java

```diff
@@ -111,7 +111,7 @@ public DirectoryReader getWriterReader(boolean flush) throws IOException {
         return writerReader;
     }
 
-    private static class FDBDirectoryMergeScheduler extends ConcurrentMergeScheduler {
+    private static class FDBDirectoryMergeScheduler extends SerialMergeScheduler {
```

Comment:
Can we make this controlled either by a `RecordContextProperty` or index option?

Reply:
✅